src/servers/middlewares/auth-router.ts

```diff
@@ -199,4 +200,55 @@ authRouter.get("/clearCookies", async (req, res) => {
   }
 })
 
+authRouter.post("/create/device-account", async (req, res) => {
+  // FIXME: try catch is too broad
+  try {
+    const ipString = isDev ? req?.ip : req?.headers["x-real-ip"]
+    const ip = parseIps(ipString)
+
+    if (ip === undefined) {
+      throw new Error("IP is not defined")
+    }
+
+    const { deviceId } = req.body
+    const user = basicAuth(req)
+
+    if (!user?.name || !user?.pass || !deviceId) {
+      return res.status(422).send({ error: "Bad input" })
+    }
+
+    const username = user.name
+    const password = user.pass
+
+    const authToken = await Auth.loginWithDevice({
+      username,
+      password,
+      ip,
+      deviceId,
+    })
+
+    if (authToken instanceof Error) {
+      // TODO open telemetry
+      return res.status(500).send({ error: authToken.message })
+    }
+
+    return res.status(200).send({
+      result: authToken,
```

Comment:
should this value be changed to just be `{ authToken }` which is what we call it in other places?